src/main/java/me/jellysquid/mods/sodium/client/model/light/data/LightDataAccess.java

```diff
@@ -71,7 +71,7 @@ protected long compute(int x, int y, int z) {
 
         // OPTIMIZE: Do not calculate lightmap data if the block is full and opaque.
         // FIX: Calculate lightmap data for light-emitting or emissive blocks, even though they are full and opaque.
-        final int lm = (fo && !em) ? 0 : block.getMixedBrightnessForBlock(world, x, y, z);
+        final int lm = (false) ? 0 : block.getMixedBrightnessForBlock(world, x, y, z);
```

Comment:
?

Reply:
Might have to revert that, I forget if this was an actual fix or just something I was testing; either way needs cleaning

Reply:
Yep, it was just for debugging